test/tinycore.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh -e
+
+# These are binaries from a mirror of
+#  http://tinycorelinux.net
+# with the following patch applied:
+# Upstream source is available http://www.tinycorelinux.net/6.x/x86/release/src/
+#BASE_URL="http://www.tinycorelinux.net/"
+
+BASE_URL="http://distro.ibiblio.org/tinycorelinux/"
```

Comment:
Is this temporary or do we expect `ibiblio.org` to be more reliably available than `tinycorelinux.net`?